src/App.tsx

```diff
@@ -17,18 +71,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filter={filter}
+                handleSetFilter={handleSetFilter}
+                handleClearInput={handleClearInput}
```

Comment:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
Same comment in other places